js/views/stock.js

```diff
@@ -1 +1,41 @@
-// define your StockView here
+var StockView = function(page){
+	this.view = document.querySelector(page);
+	this.initialize();
+	this.updateShares();
+
+}
+
+StockView.prototype.initialize = function(){
```

Comment:
For Stock, you wrote:
```
Stock.prototype = {
    method1: function(){},
    method2: function(){}
}
```
...whereas here you're writing `StockView.prototype` a bunch of times. Pick one or the other!